actions/helixUtils.js

```diff
@@ -148,10 +159,13 @@ class HelixUtils {
                 await this.bulkJobStatus(jobName, operation, repo, bulkPreviewStatus, retryAttempt + 1);
             } else if (response.ok) {
                 const jobStatusJson = await response.json();
-                logger.info(`jobStatusJson ${JSON.stringify(jobStatusJson)}`);
-                logger.info(`${operation} progress ${JSON.stringify(jobStatusJson.progress)}`);
+                // logger.info(`jobStatusJson ${JSON.stringify(jobStatusJson)}`);
+                // logger.info(`${operation} progress ${JSON.stringify(jobStatusJson.progress)}`);
                 jobStatusJson.data?.resources?.forEach((rs) => {
-                    bulkPreviewStatus[rs.path] = { success: JOB_STATUS_CODES.includes(rs.status), resourcePath: rs?.resourcePath, responseCode: rs.status };
+                    logger.info(`RS sourcein here: ${JSON.stringify(rs?.source?.name)}`);
```

Comment:
JSON.stringify is expensive. I would not suggest using this at info level. This is used at few other places in this PR.

Reply:
I still see a few usages of it
example:
https://github.com/adobecom/graybox-io/pull/11/files#diff-13937d3a10cb9be0c07e932645d50201d8570cf4c57c7db05c0036178057f6ecR156-R159

Reply:
Updated these Sunil. Thanks!

Reply:
@arshadparwaiz Note that you can keep those in if needed. Use `logger.debug(..)` instead. The logger defaults to info logs but you can pass in the log level in `getAioLogger()` before using it.